devtools-ci-index.yaml

```diff
@@ -202,6 +203,12 @@
             ci_cmd: '/bin/bash cico_run_tests.sh'
             timeout: '20m'
         - '{ci_project}-{git_repo}':
```

Comment:
@aslakknutsen you were right. Of course one has to use the `{sub_job}` in the name of the job or otherwise no job template cannot be found. I don't know why I realize this only now. Fixed in 77a4c18. Thank you.